drizzle/0004_rare_lord_tyger.sql

```diff
@@ -0,0 +1,10 @@
+ALTER TABLE `evals` ADD `tags` text;--> statement-breakpoint
+CREATE INDEX `datasets_created_at_idx` ON `datasets` (`created_at`);--> statement-breakpoint
+CREATE INDEX `evals_created_at_idx` ON `evals` (`created_at`);--> statement-breakpoint
+CREATE INDEX `evals_author_idx` ON `evals` (`author`);--> statement-breakpoint
+CREATE INDEX `evals_tags_idx` ON `evals` (`tags`);--> statement-breakpoint
```

Comment:
Dunno how this is going to perform. I think you need to do a json_extract here.

Reply:
I'll need to do some testing though to see if the index is used all the time or ever...

Reply:
NM, that doesn't work, you have to extract keys to make the index work properly..

<img width="902" alt="image" src="https://github.com/user-attachments/assets/483e8e91-75f5-48b1-8bea-1475e35ed156">


<img width="902" alt="image" src="https://github.com/user-attachments/assets/c4b9cc53-dcc9-4605-938a-a8169e88813e">